src/bootstrap/format.rs

```diff
@@ -37,6 +37,103 @@ fn rustfmt(src: &Path, rustfmt: &Path, paths: &[PathBuf], check: bool) -> impl F
     }
 }
 
+fn verify_timestamp(build: &Builder<'_>) -> bool {
```

Comment:
timestamp doesn't seem like the best name since it verifies the version, not the time.